__tests__/app.test.js

```diff
@@ -130,3 +130,48 @@ describe('Checks for valid ID returns the correct Article', () =>{
     })
 
 })
+describe('Checks all Articles for the NC news',() =>{
+    test('Testing it returns all articles with the following properties author title, article_id, topic,created_at,votes, article_img_url, comment_count',() =>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) =>{
+            body.article.forEach((article) =>{
```

Comment:
If you're going to forEach you need to check the length first.
If the return was an empty array it would cause this test to pass.